api/src/pcapi/routes/pro/offerers.py

```diff
@@ -84,6 +84,8 @@ def list_educational_offerers(
 @spectree_serialize(response_model=offerers_serialize.GetOffererResponseModel, api=blueprint.pro_private_schema)
 def get_offerer(offerer_id: int) -> offerers_serialize.GetOffererResponseModel:
     check_user_has_access_to_offerer(current_user, offerer_id)
+    # dummy change to start ci
+    print("Hello World!")
```

Comment:
possible d'avoir un peu de verbose pour savoir pourquoi c'est nécessaire ?

Reply:
Dans le nom du commit (commit to delete)
C'est un changement pour déclencher la ci, afin de vérifier que les tests failent pas etc..
Quand ce sera validé je vais drop le commit